src/Console/CommandScanner.php

```diff
@@ -64,19 +65,15 @@ public function scanCore(): array
     protected function inflectCommandNames(array $commands): array
     {
         foreach ($commands as $i => $command) {
-            $command['name'] = str_replace('_', ' ', $command['name']);
-            $command['fullName'] = str_replace('_', ' ', $command['fullName']);
-            $commands[$i] = $command;
-
-            // Maintain backwards compatibility for schema_cache as it is likely
-            // hooked up to people's deploy tooling
-            if (strpos($command['name'], 'schemacache') === 0) {
-                $compat = [
-                    'name' => str_replace('schemacache', 'schema_cache', $command['name']),
-                    'fullName' => str_replace('schemacache', 'schema_cache', $command['fullName']),
-                ];
-                $commands[] = $compat + $command;
+            $file = new SplFileInfo($command['file']);
```

Comment:
Why do you need to use `SplFileInfo`? If `$command['file']` is full file path just use string functions to get the filename.